packages/databricks-sdk-js/src/config/AzureClientSecretCredentials.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import {
+    azureEnsureWorkspaceUrl,
+    getAzureEnvironment,
+    getAzureLoginAppId,
+} from "./Azure";
+import {
+    AuthType,
+    Config,
+    CredentialProvider,
+    RequestVisitor,
+    Headers,
+} from "./Config";
+import {Issuer, TokenSet} from "openid-client";
```

Comment:
please re-implement it without external dependencies as in Go / Python / Java SDKs - it's just resolving a couple of URLs.